src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -7,7 +7,7 @@ import React, { useEffect, useRef } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import GlobalConstants from 'GlobalConstants';
-import { useQueryParams } from 'helpers/queryParams';
+import { combineQueryParams, getQueryParamString, useQueryParams } from 'helpers/queryParams';
```

Comment:
```suggestion
import { getQueryParamString, useQueryParams } from 'helpers/queryParams';
```
(`no-unused-vars` lint error)